docs/index.md

```diff
@@ -0,0 +1,18 @@
+![PowSyBl Logo](_static/logos/logo_lfe_powsybl.svg)
```

Comment:
I find it a bit oppressive this big colourful logo! :smile: but the main problem is to have something uniform across the repositories. We should have the same kind of home page for all of them, and so far some are including it, some not